.bootstrap.mk

```diff
@@ -50,53 +50,75 @@ SHELL       := bash
 override ROOT := $(shell path="$(CURDIR)"; while [[ "$${path}" != "/" \
  && ! -f "$${path}/.bootstrap.mk" ]]; do path="$${path%/*}"; done; echo "$${path}")
 
+## Built-in
+
 # A generic help message that parses the available targets, and lists each one
 # that has a comment on the same line with a ## prefix.
 help: ## Display this help
-	readonly pad=25; \
-	targets=(); \
-	pattern='[^:]+:[^#]*## .*'; \
-	makefiles=($(MAKEFILE_LIST)); \
-	\
-	echo; \
-	echo -e "\033[0;33mUsage:\033[0m\n\
-		make [flags...] [target...] [options...]\n\
-	\n\
-	\033[0;33mFlags:\033[0m\n\
-		See \033[1;30mmake --help\033[0m\n\
-	\n\
-	\033[0;33mTargets:\033[0m"; \
-	\
-	for mk in "$${makefiles[@]}"; do \
-		while read -r line; do \
-			if [[ "$${line}" =~ $${pattern} ]]; then \
-				targets+=("$${line}"); \
-			fi \
-		done < "$${mk}"; \
-	done; \
-	\
-	readarray -t targets < <(for t in "$${targets[@]}"; do echo "$$t"; done | sort -d); \
-	\
-	for target in "$${targets[@]}"; do \
-		printf "    \033[0;32m%-$${pad}s\033[0m %s\n" "$${target%%:*}" "$${target##*## }"; \
-	done; \
-	\
+	readonly pad=$$(printf "%0.1s" "-"{1..25})
```

Comment:
Not sure yet whether I like the lines. Makes it less readable to my eyes. Let me think about it.

Reply:
Y see what you mean. I can think of two options right now:
- Change appearance of lines: use another symbol, less contrast color (gray) or both.
- Include an option to specify the help style, as far as there are several of the already.

Reply:
Less contrast sounds like the best option.
Having the option to customize the styling is maybe a bit too much.

Reply:
Here how it looks now in my environment:

![image](https://user-images.githubusercontent.com/1475583/109484526-7145dc00-7a78-11eb-8528-e666154fc23c.png)